packages/encryption/src/lib/encryption.ts

```diff
@@ -283,7 +285,12 @@ export const decryptZip = async (
     },
   });
 
-  if (!paramsIsSafe) return;
+  if (!paramsIsSafe){
+    throwError({
```

Comment:
I think this should be the default approach going forward instead of returning `undefined`. I've seen the later at multiple places we should updates those too prob in future